config/.jekyll-labels.yml

```diff
@@ -5,3 +5,4 @@
 "type: Informational": this?.new?.type == "Informational"
 "type: Meta": this?.new?.type == "Meta"
 "type: EIP1 (Process)": this?.old?.status == "Living" || this?.old?.eip == "<to be assigned>"
+"waiting: EIP number": '!isNaN(this?.new?.eip)'
```

Comment:
Why wouldn't an EIP have a number w.r.t. to jekyll? During local site generation before the number is assigned?

Reply:
Yes, if the *new* EIP Number is something like `<to be assigned>`.